spec/controllers/expenses_additional_controller_spec.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe ExpensesAdditionalController, type: :controller do
+  let(:step) { assigns(:step) }
+  let(:step_class) { ExpensesAdditional }
+  before {}
```

Comment:
oh I see -- we are setting in each spec. In other controller specs we've been doing this via a `before` action. I generally don't love `before` but it kinda makes sense here since they all need it